### DISCUSSION TITLE
中国移动云盘异常

### DISCUSSION THREAD

**Discussion by iamhcfhsgl** [AlistGo/alist#7460] [Q&A]
中国移动云盘改版后没办法直接搜索到getdisk，在其他地方也能找到Authorization，能正常挂载，但还有一个很致命的问题，在Alist上传的文件，确实上传成功了，在网页端查看该账号的剩余容量，可以看出，文件确实是上传进去了，也占用了空间，但是在网页端的根目录根本找不到文件，同理，在网页端上传的和创建的文件夹在Alist那也看不见，这是出了什么bug？

问题找到了，网页端被显示成新个人云，但是我的账号同时存在个人云和新个人云两个类型，挂载新个人云和个人云都能成功，但我的数据都存在个人云里，没办法迁移在移动网盘的客户端上的新个人云，也没办法查看，以前只能挂载个人云，数据都存在那，客户端显示空间被占用了，却找不到文件，虽然问题找到了，但数据怎么迁移？快100gb，能同时添加两个驱动器直接移动吗？好像不行

是移动的锅，自己升级系统，把我的账号从个人云转到新个人云，却没完整迁移数据，只有之前通过alist上传的数据没被迁移，在客户端上传的都正常迁移了，应该是alist上传的数据没有被标记，导致出了bug。

**Comment by aaaleileilei** [comment]
不知道有没有什么办法，我用网页登陆进去是个人云，也能找到getdisk，但Alist无法挂载个人云，只能挂载新个人云，然后手机端和网页端显示的都是个人云，导致数据不互通。

**Comment by mondayice** [comment]
> 中国移动云盘改版后没办法直接搜索到getdisk，在其他地方也能找到Authorization，能正常挂载，但还有一个很致命的问题，在Alist上传的文件，确实上传成功了，在网页端查看该账号的剩余容量，可以看出，文件确实是上传进去了，也占用了空间，但是在网页端的根目录根本找不到文件，同理，在网页端上传的和创建的文件夹在Alist那也看不见，这是出了什么bug？
> 
> 问题找到了，网页端被显示成新个人云，但是我的账号同时存在个人云和新个人云两个类型，挂载新个人云和个人云都能成功，但我的数据都存在个人云里，没办法迁移在移动网盘的客户端上的新个人云，也没办法查看，以前只能挂载个人云，数据都存在那，客户端显示空间被占用了，却找不到文件，虽然问题找到了，但数据怎么迁移？快100gb，能同时添加两个驱动器直接移动吗？好像不行
> 
> 是移动的锅，自己升级系统，把我的账号从个人云转到新个人云，却没完整迁移数据，只有之前通过alist上传的数据没被迁移，在客户端上传的都正常迁移了，应该是alist上传的数据没有被标记，导致出了bug。

2025年2月18日 21:05:22
目前我也是这个问题，在Alist上传的文件和创建文件夹在APP端和Web端都不可见，反之也不可见。我是老账号。
